src/core/lightning/index.ts

```diff
@@ -313,6 +314,9 @@ export const LightningMixin = (superclass) =>
             type: "paid-invoice",
           })
 
+          const eventName = lnPaymentStatusEvent(id)
+          pubsub.publish(eventName, { status: "PAID" })
```

Comment:
Do we need a `publish` for successful `withdrawal` payments as well (since this one is for `onUs`)?

Reply:
uhmm maybe for hodl invoices but that is an edge case, in general, when you pay an invoice the status response is immediate